docker/elasticsearch/elasticsearch.yml

```diff
@@ -26,4 +30,4 @@ xpack.graph.enabled: false
 xpack.watcher.enabled: false
 
 #Accept all requests from localhost, dashboard and kibana
-#network.host: [127.0.0.1, localhost, dashboard, kibana, nginx]
+network.host: [127.0.0.1, localhost]
```

Comment:
@fxprunayre please note that the requests to elasticsearch are coming from `dashboard` and `kibana` addresses, **not** from localhost.